pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -78,6 +78,9 @@
 public class SegmentColumnarIndexCreator implements SegmentCreator {
   // TODO Refactor class name to match interface name
   private static final Logger LOGGER = LoggerFactory.getLogger(SegmentColumnarIndexCreator.class);
+  // Allow at most 512 characters for the metadata property
+  private static final int METADATA_PROPERTY_LENGTH_LIMIT = 512;
```

Comment:
shall this align with `FieldSpec.DEFAULT_MAX_LENGTH`?

Reply:
They are independent, where the `maxLength` is configurable in the `FieldSpec` to allow ingesting long strings, but the property length limit is fixed. Also, the property length limit also applies to the BYTES column.

Reply:
I see. Thanks